src/index.js

```diff
@@ -35,7 +35,7 @@ export default class SecureLS {
       constants.EncrytionTypes.BASE64;
     this.config.encryptionSecret = config.encryptionSecret;
 
-    this.ls = localStorage;
+    this.ls = config.storageType || localStorage;
```

Comment:
This is wrong! It should be an object and not a string.
The code should be:

```
this.config.storageType = config.storageType || 'localStorage';

this.ls = config.storageType || localStorage;
if (config.storageType && config.storageType === 'sessionStorage') {
  this.storage = sessionStorage;
} else {
  this.storage = localStorage;
}
```

Reply:
I'm a bit confused, you said "It should be an object and not a string", but in your example `this.config.storageType` is set to a string and not an object 🤔 . Also setting a custom storage type (anything besides sessionStorage/localStorage) is not possible this way.